### PR TITLE
Add sam optimized ViT and MLP-Mixer checkpoints

### PR DESCRIPTION
**Pull request by xiangning-chen**

### CODE REVIEW

**Comment by xiangning-chen** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Vision Transformer and MLP-Mixer Architectures
 
+**Update (2.7.2021)**: Added the "When Vision Transformers Outperform ResNets..." paper, and SAM optimized ViT and MLP-Mixer checkpoints.
```
Comment (andsteing) [inline README.md:3]: Could you break lines at 80 characters?
(We only have lines with links & tables running over the character limit)

File: README.md [hunk 2]
```diff
@@ -181,6 +186,8 @@ We provide models pre-trained on imagenet21k for the following architectures:
 ViT-B/16, ViT-B/32, ViT-L/16 and ViT-L/32. We  provide the same models
 pre-trained on imagenet21k *and* fine-tuned on imagenet2012.
 
+**Update (2.7.2021)**: We added the ViT models trained with [SAM](https://arxiv.org/abs/2010.01412) optimizer on imageNet2012 (with basic Inception-style preprocessing). The resultant ViTs outperform ResNets of similar size and throughput without large-scale pretraining or strong data augmentations. They also possess more perceptive attention maps. To use those models, you can simply replace the model path in [`vit_jax.ipynb`] with [`gs://vit_models/sam`].
```
Comment (andsteing) [inline README.md:189]: Maybe introduce the "SAM" abbreviation further up (or here)?

File: README.md [hunk 3]
```diff
@@ -445,6 +456,15 @@ And finally execute one of the commands mentioned in the section
   journal={arXiv preprint arXiv:2106.TODO},
   year={2021}
 }
+
+@misc{chen2021vision,
```
Comment (andsteing) [inline README.md:460]: Maybe `chen2021sam` or `chen2021outperform`?
(Not sure what's customary, but I find it useful to have the most salient word in the reference)

File: README.md [hunk 4]
```diff
@@ -291,6 +298,10 @@ For installation follow [the same steps](#installation) as above.
 
 ### Available Mixer models
 
+[`gs://mixer_models/sam`]: https://console.cloud.google.com/storage/mixer_models/sam/
```
Comment (andsteing) [inline README.md:301]: Is this needed multiple times?
Usually we put it after the first paragraph using the reference.

File: README.md (hunk 2 above)
Comment (andsteing) [inline README.md:189]: Nit: We say "ImageNet" in the main text (maybe you could also fix line 187 👍 )

Comment (andsteing) [inline README.md:189]: Nit: We say "pre-train" (unfortunately not matching your title...)

File: README.md [hunk 5]
```diff
@@ -291,6 +298,10 @@ For installation follow [the same steps](#installation) as above.
...
+
+**Update (2.7.2021)**: We added the MLP-Mixer models trained with [SAM](https://arxiv.org/abs/2010.01412) on imageNet2012 without strong augmentations ([`gs://mixer_models/sam`]). The loss landscapes become much smoother, and we found that the activate neurons for the first few layers decrease dramatically after SAM, indicating the potential redundency of image patches.
```
Comment (andsteing) [inline README.md:303]: "active"? "activated"?

**Review by andsteing — CHANGES_REQUESTED** [review]
Thanks for adding these! Looks good, added some nits mostly about formatting...

**Review by andsteing — APPROVED** [review]

**Pull request merged**